agr_java_core/src/main/java/org/alliancegenome/core/variant/service/AlleleVariantIndexService.java

```diff
@@ -81,69 +81,64 @@
 
             }
         }
-        System.out.println("TOTAL HITS:"+sr.getHits().getTotalHits());
-        System.out.println("Allele Variant Sequences:" +avs.size());
-
+        
+        log.debug("TOTAL HITS:" + searchResponce.getHits().getTotalHits());
+        log.debug("Allele Variant Sequences:" + avsList.size());
```

Comment:
System.out.println's do not show up in the logs so a `log.debug` or a `log.info` is much better to use.